addons/project/security/project_security.xml

```diff
@@ -114,5 +114,12 @@
         <field name="groups" eval="[(4, ref('base.group_portal'))]"/>
     </record>
 
+    <record id="access_project_portal_share_rule" model="ir.rule">
+        <field name="name">Project/Task: Allow share tasks</field>
+        <field name="model_id" ref="portal.model_portal_share"/>
+        <field name="domain_force">[('res_model', '=', project.task')]</field>
```

Comment:
Is there a global record rule [(1, =, 1)] for the partner_manager group?

Reply:
There are currently no rules.
Ok, I understand what you are saying, with this change, the general [(1, =, 1)] rule should also be added in `portal` addon, right?.